diff -urN squidguard_AU-0.3.0.orig/squidguard_AU.sh squidguard_AU-0.3.0/squidguard_AU.sh
--- squidguard_AU-0.3.0.orig/squidguard_AU.sh	2014-09-25 20:08:42.000000000 +0200
+++ squidguard_AU-0.3.0/squidguard_AU.sh	2014-10-02 14:16:20.221430170 +0200
@@ -51,5 +51,7 @@
 chown -R ${SQUIDUSER}:${SQUIDUSER} ${DATABASE}
 chown -R ${SQUIDUSER}:${SQUIDUSER} ${LOGFILE}
 
-/etc/init.d/squid reload
+echo -e "$MESG Reloading squid configuration ..."
+systemctl reload squid
+
 exit 0
